### PR TITLE
Disable idle timer while video calling

### PR DESCRIPTION
**Pull request by daehn**
## What's new in this PR?

* Disable idle timer while video calling.

### This PR depends on:

- [x] Some final testing still has to be done.

### CODE REVIEW

File: Wire-iOS Tests/CallInfoTestFixture.swift [hunk 1]
```diff
@@ -456,7 +477,8 @@ struct CallInfoTestFixture {
             isConstantBitRate: false,
             title: otherUser.displayName,
             isVideoCall: true,
-            variant: .light
+            variant: .light,
+            disableIdleTimer: false
```
Comment (typfel) [inline Wire-iOS Tests/CallInfoTestFixture.swift]: Shouldn't `disableIdleTimer` be true in this case?

File: Wire-iOS Tests/Mocks/MockCallInfoViewControllerInput.swift [hunk 2]
```diff
@@ -34,4 +34,5 @@ struct MockCallInfoViewControllerInput: CallInfoViewControllerInput {
     var title: String
     var isVideoCall: Bool
     var variant: ColorSchemeVariant
+    var disableIdleTimer: Bool
```
Comment (typfel) [inline Wire-iOS Tests/Mocks/MockCallInfoViewControllerInput.swift]: I guess this should be added to the Equality function?

File: Wire-iOS Tests/CallInfoTestFixture.swift (hunk 1 above)
Comment (daehn) [reply Wire-iOS Tests/CallInfoTestFixture.swift]: I just filled it in with false everywhere in the tests as it shouldn't have any effect there anyways.

Comment (daehn) [reply Wire-iOS Tests/CallInfoTestFixture.swift]: Could maybe also generally add the implementation of this as a protocol extension.

**Comment by daehn** [comment]
@typfel updated!

**Pull request merged**